cafebabel/templates/articles/translations/create.html

```diff
@@ -8,24 +9,25 @@
 {% block body_class %}article-page-edit{% endblock %}
 {% block body %}
 
-<form method=post action=.>
-  <input type=hidden name=original value="{{ article.id }}">
-  {% if article.has_image %}
-    <img src="{{ article.image_url }}" alt="Image for {{ article.title }}">
-  {% endif %}
-  <h1 class="edit active">
-    <label for=title>Title</label>
-    <input id=title name=title required placeholder="Translation in {{ language }} for article “{{ article.title }}”">
-  </h1>
-  <div class=textarea>
-    <label for=summary>Teaser</label>
-    <textarea id=summary name=summary required></textarea>
-  </div>
-  <div class=textarea>
-    <label for=body>Body</label>
-    <textarea id=body name=body required></textarea>
-    {% include "articles/markdown-helper.html" %}
-  </div>
-  <input type=submit value="Create">
-</form>
-{% endblock %}
+  <form method=post action=.>
+    <input type=hidden name=original value="{{ article.id }}">
+    {% if article.has_image %}
+      <img src="{{ article.image_url }}" alt="Image for {{ article.title }}">
+    {% endif %}
+    <h1 class="edit active">
+      <label for=title>Title</label>
+      <input id=title name=title required placeholder="Translation in {{ language }} for article “{{ article.title }}”">
+    </h1>
+    <div class=textarea>
+      <label for=summary>Teaser</label>
+      <textarea id=summary name=summary required></textarea>
+    </div>
+    <div class=textarea>
+      <label for=body>Body</label>
+      <textarea id=body name=body required></textarea>
+      {% include "articles/markdown-helper.html" %}
+    </div>
+    {{ macros.tags_fields(translation) }}
```

Comment:
Here i screwed up, I juste add this line actually: `{{ macros.tags_fields(translation) }}`. The rest stayed the same @davidbgk